src/packages.config

```diff
@@ -1,38 +1,38 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Cake.Core" version="0.19.4" targetFramework="net451" developmentDependency="true" />
+  <package id="Cake.Testing" version="0.19.4" targetFramework="net451" developmentDependency="true" />
   <package id="ini-parser" version="2.3.0" targetFramework="net451" />
-  <package id="Microsoft.VisualStudio.CoreUtility" version="14.2.25123" targetFramework="net451" />
-  <package id="Microsoft.VisualStudio.Imaging" version="14.2.25123" targetFramework="net451" />
+  <package id="Microsoft.VisualStudio.CoreUtility" version="15.0.26201" targetFramework="net46" />
+  <package id="Microsoft.VisualStudio.Imaging" version="15.0.26201" targetFramework="net46" />
   <package id="Microsoft.VisualStudio.OLE.Interop" version="7.10.6070" targetFramework="net451" />
-  <package id="Microsoft.VisualStudio.Shell.14.0" version="14.2.25123" targetFramework="net451" />
+  <package id="Microsoft.VisualStudio.Shell.14.0" version="14.3.25407" targetFramework="net46" />
   <package id="Microsoft.VisualStudio.Shell.Immutable.10.0" version="10.0.30319" targetFramework="net451" />
```

Comment:
Thanks @mholo65 ! Every time I give MS the benefit of the doubt about framework re-targets, I get proven wrong 😢 

*Unrelatedly, there was a really cool VS extension for migrating target frameworks for a whole solution, but doesn't support 2017 :( *